lib/providers/quoters/RoutingApiQuoter.ts

```diff
@@ -40,6 +40,24 @@ export class RoutingApiQuoter implements Quoter {
       }
       log.error(e, 'RoutingApiQuoterErr');
       metrics.putMetric(`RoutingApiQuoterErr`, 1);
+
+      // We want to ensure that we throw all non-404 errors
+      // to ensure that the client will know the request is retryable.
+      // We include 429's in the retryable errors because a 429 would
+      // indicate that the Routing API was being rate-limited and a subsequent
+      // retry may succeed.
```

Comment:
nit but isn't the new logic to throw 500 on 429 so the client doesn't retry?

Reply:
So on 500 returned from URA will trigger retries. When we 429 from routing API we want a retry.